Training/configs/training_v1.yaml

```diff
@@ -0,0 +1,285 @@
+Setup:
+    n_tau                :  500
+    start_dataset        :  0
+    end_dataset          :  9999999999999 #std::numeric_limits<Long64_t>::max()
+    n_inner_cells        :  11
+    inner_cell_size      :  0.02
+    n_outer_cells        :  21
+    outer_cell_size      :  0.05
+    n_threads            :  1
+    n_fe_tau             :  43
+    n_pf_el              :  22
+    n_pf_mu              :  23
+    n_pf_chHad           :  27
+    n_pf_nHad            :  7
+    n_pf_gamma           :  23
+    n_ele                :  37
+    n_muon               :  37
```

Comment:
`n_*` variables should be computed automatically based on the list of active features